mesonbuild/interpreter.py

```diff
@@ -96,6 +97,19 @@ def disabled_method(self, args, kwargs):
     def auto_method(self, args, kwargs):
         return self.held_object.is_auto()
 
+    @noPosargs
+    @permittedKwargs({})
+    def to_string_method(self, args, kwargs):
+        if self.held_object.is_enabled():
+            return 'enabled'
+        elif self.held_object.is_disabled():
+            return 'disabled'
+        elif self.held_object.is_auto():
+            return 'auto'
+
+        # Should never reach here
+        return 'unknown'
```

Comment:
I think you should raise an exception instead of silently returning a value user won't expect.

Reply:
Sure, I can do that.

Reply:
This still needs fixing. Since this is new functionality, this MR should add a release note snippet and update the `build-options` documentation page.

Reply:
ping